client/generated_snapshot.go

```diff
@@ -7,7 +7,7 @@ const (
 type Snapshot struct {
 	Resource `yaml:"-"`
 
-	Children map[string]string `json:"children,omitempty" yaml:"children,omitempty"`
+	Children map[string]interface{} `json:"children,omitempty" yaml:"children,omitempty"`
```

Comment:
It should be `map[string]bool` here. This wrongly generated code means you need to re-check the code in the `api/model.go`

Reply:
It doesn't seem like `go-rancher` can actually generate a `map[string]bool` without changes to the code of `go-rancher`. The code here seems to indicate that it'll only handle `map[string]` as `map[string]string` and a `map` of any other type as `map[string]interface{}`: https://github.com/niusmallnan/go-rancher/blob/a114f4efa63bbff39fcd8a7b6982b83e29ad36f5/generator/generator.go#L80